docker/run_test.sh

```diff
@@ -75,5 +75,5 @@ $NOSE socorro -s
 # Collect static and then test webapp
 pushd webapp-django
 python manage.py collectstatic --noinput
-./bin/ci.sh
+./bin/ci.sh --docker
```

Comment:
I had forgotten which way I did this flag and not having it here was causing the tests to fail because it would look for a `socorro-virtualenv` thing which we don't do in the docker environment.